app/views/ati_network/showcase/_hero.html.erb

```diff
@@ -0,0 +1,62 @@
+<section class="hero-section">
+  <%= image_tag('ati_network/showcase/worldmap-pale.svg', alt: _('World map'), :class => "hero-section-image") %>
+
+  <div class="hero-content">
+    <div class="row hero-main-content">
+      <p class="hero-preheading">Access to Information</p>
+
+      <h1>Around the World</h1>
+
+      <p class="hero-subtitle">
+        This site is part of a worldwide ecosystem of platforms helping people request public information around the world.</p>
+    </div>
+
+    <div class="row hero-list-sites-section">
+      <h2>This site and many others are powered by Alaveteli, a service by <%= image_tag('ati_network/showcase/logo-mysociety.png', alt: _('mySociety'), :class => "hero-mysoc-logo") %></h2>
```

Comment:
Can we:

- [ ] Make the logo a link to https://www.mysociety.org/
- [ ] Make it inline with the text rather than block, and about the same size as the text so that it looks like its part of the sentence rather than a separate element